contracts/AutomationCounter.sol

```diff
@@ -1,9 +1,9 @@
 // SPDX-License-Identifier: MIT
-pragma solidity ^0.8.7;
+pragma solidity >=0.8.6;
```

Comment:
This pragma change would mean that we want this contract to compile even for v0.9. However, note that there may be breaking changes (the contracts depended on are versioned into new folders for new minor versions.  Eg we are now depending on src/v0.8/../.. which is different from v0.7 and will be different from v0.9 etc). The original caret based semver is right as it only compiles >0.8 and <0.9